tests/tck-build-logic/src/main/groovy/org/graalvm/internal/common/MetadataTest.groovy

```diff
@@ -0,0 +1,147 @@
+package org.graalvm.internal.common
+
+import org.graalvm.internal.tck.Utils
+import org.graalvm.internal.tck.harness.MetadataLookupLogic
+
+import java.nio.file.Path
+
+import static org.graalvm.internal.tck.TestUtils.metadataRoot
+import static org.graalvm.internal.tck.TestUtils.testRoot
+
+class MetadataTest {
```

Comment:
This looks to me more like `MetadataDescriptor`, since it doesn't only concern tests.